source/API/Extended.cpp

```diff
@@ -356,8 +356,30 @@ Extended::getBundle(const Type::Trytes&) const {
 }
 
 getTransfersResponse
-Extended::getTransfers(const Type::Trytes&, int, int, int, bool) const {
-  return { {}, 0 };
+Extended::getTransfers(const Type::Trytes& seed, int security, int start, int end,
+                       bool inclusionStates) const {
+  Utils::StopWatch stopWatch;
+
+  // Validate the seed
+  if ((!Type::Seed::isValidSeed(seed))) {
+    throw Errors::IllegalState("Invalid Seed");
+  }
+
+  // Validate the security level
+  if (security < 1 || security > 3) {
+    throw Errors::IllegalState("Invalid Security Level");
+  }
+
+  // If start value bigger than end, return error
+  // or if difference between end and start is bigger than 500 keys
+  if (start > end || end > (start + 500)) {
+    throw Errors::IllegalState("Invalid inputs provided");
+  }
+
+  auto gnr     = getNewAddresses(seed, security, start, false, end, true);
```

Comment:
Parameters are not in the correct order.

`getNewAddressesResponse
Extended::getNewAddresses(const Type::Trytes& seed, const uint32_t& index, const int32_t& security,
                          bool checksum, const int32_t& total, bool returnAll) const`

Reply:
good catch